### PR TITLE
Updating AI quickstarts to reflect new sample separation of OpenAI and Azure OpenAI

### PR DESCRIPTION
**Pull request by jmatthiesen**
## Summary

We're simplifying the SDKs used in our AI quickstarts to focus on using Semantic Kernel, and demonstrating how to use either OpenAI or Azure OpenAI as the host service. These updates accompany a samples update tracked in the following PR:

https://github.com/dotnet/ai-samples/pull/71

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/ai/get-started/dotnet-ai-overview.md](https://github.com/dotnet/docs/blob/1f70b9c56ec722ee79713ae72afaf21700a1c547/docs/ai/get-started/dotnet-ai-overview.md) | [Develop .NET applications with AI features](https://review.learn.microsoft.com/en-us/dotnet/ai/get-started/dotnet-ai-overview?branch=pr-en-us-41620) |
| [docs/ai/quickstarts/get-started-openai.md](https://github.com/dotnet/docs/blob/1f70b9c56ec722ee79713ae72afaf21700a1c547/docs/ai/quickstarts/get-started-openai.md) | [CustomerIntent: As a .NET developer new to AI, I want deploy and use sample code to interact to learn from the sample code.](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/get-started-openai?branch=pr-en-us-41620) |
| [docs/ai/quickstarts/quickstart-ai-chat-with-data.md](https://github.com/dotnet/docs/blob/1f70b9c56ec722ee79713ae72afaf21700a1c547/docs/ai/quickstarts/quickstart-ai-chat-with-data.md) | [Get insight about your data from a .NET AI chat app](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-ai-chat-with-data?branch=pr-en-us-41620) |
| [docs/ai/quickstarts/quickstart-azure-openai-tool.md](https://github.com/dotnet/docs/blob/1f70b9c56ec722ee79713ae72afaf21700a1c547/docs/ai/quickstarts/quickstart-azure-openai-tool.md) | [Quickstart - Extend OpenAI using Tools and execute a local Function with .NET ](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-azure-openai-tool?branch=pr-en-us-41620) |
| [docs/ai/quickstarts/quickstart-openai-generate-images.md](https://github.com/dotnet/docs/blob/1f70b9c56ec722ee79713ae72afaf21700a1c547/docs/ai/quickstarts/quickstart-openai-generate-images.md) | [CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact to learn how to generate images from the sample code.](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-openai-generate-images?branch=pr-en-us-41620) |
| [docs/ai/quickstarts/quickstart-openai-summarize-text.md](https://github.com/dotnet/docs/blob/1f70b9c56ec722ee79713ae72afaf21700a1c547/docs/ai/quickstarts/quickstart-openai-summarize-text.md) | [Summarize text using AI chat app with .NET](https://review.learn.microsoft.com/en-us/dotnet/ai/quickstarts/quickstart-openai-summarize-text?branch=pr-en-us-41620) |

### CODE REVIEW

**Comment by jmatthiesen** [comment]
Reviews are welcome now! I'm getting an error in the Markdownlint step that I can't figure out how to fix, so feedback is definitely welcome there as well.

**Review by alexwolfmsft — APPROVED** [review]

File: docs/ai/quickstarts/includes/troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+> WARNING: 'postprovision' hook failed with exit code: '1', Path: '.\infra\post-script\postprovision.ps1'. : exit code: 1
+> Execution will continue since ContinueOnError has been set to true.
+
+The script **postprovision.ps1** is executed to set the .NET user secrets used in the application. To avoid this error, manually run the script using the following PowerShell command:
+
+```powershell
+.\infra\post-script\postprovision.ps1
+```
+
+The .NET AI apps now have the user-secrets configured and they can be tested.
```
Comment (gewarren) [inline docs/ai/quickstarts/includes/troubleshoot.md:35]:
```suggestion
The .NET AI apps now have the user secrets configured and they can be tested.
```

File: docs/ai/quickstarts/quickstart-ai-chat-with-data.md [hunk 2]
```diff
@@ -1,65 +1,112 @@
...
-# Get insight about your data from an .NET Azure AI chat app
+# Get insight about your data from a .NET AI chat app
 
 <!-- markdownlint-disable MD044 -->
-:::zone target="docs" pivot="semantic-kernel"
+:::zone target="docs" pivot="openai"
 <!-- markdownlint-enable MD044 -->
 
-Get started with Semantic Kernel and the `gpt-35-turbo` model, from a simple .NET 8.0 console application. Use the AI model to get analytics and information about your previous hikes. It consists of a simple console application, running locally, that will read the file `hikes.md` and send request to an Azure OpenAI service deployed in your Azure subscription and provide the result in the console. Follow these steps to provision Azure OpenAI and learn how to use Semantic Kernel.
+Get started with AI development using a .NET 8.0 console app to connect to an OpenAI `gpt-3.5-turbo` model. You'll connect to the AI model using [Semantic Kernel](../semantic-kernel-dotnet-overview.md) to analyze hiking data and provide insights.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-ai-chat-with-data.md:19]:
```suggestion
Get started with AI development using a .NET 8 console app to connect to an OpenAI `gpt-3.5-turbo` model. You'll connect to the AI model using [Semantic Kernel](../semantic-kernel-dotnet-overview.md) to analyze hiking data and provide insights.
```

File: docs/ai/quickstarts/quickstart-ai-chat-with-data.md [hunk 3]
```diff
@@ -1,65 +1,112 @@
...
+[!INCLUDE [download-alert](includes/prerequisites-openai.md)]
 :::zone-end
 
 <!-- markdownlint-disable MD044 -->
-:::zone target="docs" pivot="azure-openai-sdk"
+:::zone target="docs" pivot="azure-openai"
 <!-- markdownlint-enable MD044 -->
 
-Get started with the .NET Azure OpenAI with a `gpt-35-turbo` model, from a simple .NET 8.0 console application. Use the AI model to get analytics and information about your previous hikes. It consists of a simple console application, running locally, that will read the file `hikes.md` and send request to an Azure OpenAI service deployed in your Azure subscription and provide the result in the console. Follow these steps to provision Azure OpenAI and learn how to use the .NET Azure OpenAI SDK.
+Get started with AI development using a .NET 8.0 console app to connect to an OpenAI `gpt-3.5-turbo` model deployed on Azure. You'll connect to the AI model using [Semantic Kernel](../semantic-kernel-dotnet-overview.md) to analyze hiking data and provide insights.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-ai-chat-with-data.md:28]:
```suggestion
Get started with AI development using a .NET 8 console app to connect to an OpenAI `gpt-3.5-turbo` model deployed on Azure. You'll connect to the AI model using [Semantic Kernel](../semantic-kernel-dotnet-overview.md) to analyze hiking data and provide insights.
```

File: docs/ai/quickstarts/quickstart-ai-chat-with-data.md [hunk 4]
```diff
@@ -1,65 +1,112 @@
...
+
+2. Use the `dotnet run` command to run the app:
 
     ```dotnetcli
     dotnet run
     ```
 
-    If you get an error message the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
+    > [!TIP]
+    > If you get an error message, the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-ai-chat-with-data.md:76]:
```suggestion
    > If you get an error message, the Azure OpenAI resources might not have finished deploying. Wait a couple of minutes and try again.
```

File: docs/ai/quickstarts/quickstart-openai-generate-images.md [hunk 5]
```diff
@@ -10,168 +10,131 @@ zone_pivot_groups: openai-library
...
+    dotnet run
+    ```
 
-```csharp
-// == Retrieve the local secrets saved during the Azure deployment ==========
-var config = new ConfigurationBuilder()
-    .AddUserSecrets<Program>()
-    .Build();
+    > [!TIP]
+    > If you get an error message, the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-openai-generate-images.md:73]:
```suggestion
    > If you get an error message, the Azure OpenAI resources might not have finished deploying. Wait a couple of minutes and try again.
```

File: docs/ai/quickstarts/quickstart-openai-generate-images.md [hunk 6]
```diff
@@ -10,168 +10,131 @@ zone_pivot_groups: openai-library
...
-
-ImageGenerationData generatedImage = response.Value.Data[0];
-if (!string.IsNullOrEmpty(generatedImage.RevisedPrompt))
-{
-    Console.WriteLine($"\n\nInput prompt automatically revised to:\n {generatedImage.RevisedPrompt}");
-}
-Console.WriteLine($"\n\nThe generated image is ready at:\n {generatedImage.Url.AbsoluteUri}");
+// Generate the image
+string imageUrl = await textToImageService.GenerateImageAsync("""
+    A postal card with an happy hiker waving and a beautiful mountain in the background.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-openai-generate-images.md:127]:
```suggestion
    A postal card with a happy hiker waving and a beautiful mountain in the background.
```

File: docs/ai/quickstarts/quickstart-azure-openai-tool.md [hunk 7]
```diff
@@ -1,66 +1,117 @@
...
+
+2. Use the `dotnet run` command to run the app:
 
     ```dotnetcli
     dotnet run
     ```
 
-    If you get an error message the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
+    > [!TIP]
+    > If you get an error message, the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-azure-openai-tool.md:78]:
```suggestion
    > If you get an error message, the Azure OpenAI resources might not have finished deploying. Wait a couple of minutes and try again.
```

File: docs/ai/quickstarts/quickstart-azure-openai-tool.md [hunk 8]
```diff
@@ -1,66 +1,117 @@
...
+
 <!-- markdownlint-disable MD044 -->
-:::zone target="docs" pivot="semantic-kernel"
-<!-- markdownlint-enable MD044 -->
+:::zone target="docs" pivot="openai"
 
-## Understand the code
+The application uses the [`Microsoft.SemanticKernel`](https://www.nuget.org/packages/Microsoft.SemanticKernel) package to send and receive requests to the OpenAI service.
+
+The entire application is contained within the **Program.cs** file. The first several lines of code set configuration values and gets the OpenAI Key that was previously set using the `dotnet user-secrets` command.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-azure-openai-tool.md:90]:
```suggestion
The entire application is contained within the **Program.cs** file. The first several lines of code set configuration values and get the OpenAI Key that was previously set using the `dotnet user-secrets` command.
```

File: docs/ai/quickstarts/quickstart-azure-openai-tool.md [hunk 9]
```diff
@@ -82,219 +133,70 @@ Kernel kernel = b
     .Build();
 ```
 
-The function's `ImportPluginFromFunctions` and `CreateFromMethod` are used to define the local function that will be called by the model.
+:::zone-end
+
+The function's `ImportPluginFromFunctions` and `CreateFromMethod` define the local function that will be called by the model.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-azure-openai-tool.md:138]:
```suggestion
The functions `ImportPluginFromFunctions` and `CreateFromMethod` define the local function that will be called by the model.
```

File: docs/ai/quickstarts/quickstart-azure-openai-tool.md [hunk 10]
```diff
@@ -82,219 +133,70 @@ Kernel kernel = b
...
+Once the `kernel` client is created, the code uses a system prompt to provide context and influence the completion tone and content. Note how the weather is emphasized in the system prompt.
 
 ```csharp
 ChatHistory chatHistory = new("""
-    You are a hiking enthusiast who helps people discover fun hikes in their area. You are upbeat and friendly.
-    A good weather is important for a good hike. Only make recommendations if the weather is good or if people insist.
-    You introduce yourself when first saying hello. When helping people out, you always ask them 
-    for this information to inform the hiking recommendation you provide:
+    You are a hiking enthusiast who helps people discover fun hikes in their area.
+    You are upbeat and friendly. A good weather is important for a good hike.
```
Comment (gewarren) [inline docs/ai/quickstarts/quickstart-azure-openai-tool.md:159]:
```suggestion
    You are upbeat and friendly. Good weather is important for a good hike. 
```

File: docs/ai/quickstarts/get-started-openai.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+# CustomerIntent: As a .NET developer new to AI, I want deploy and use sample code to interact to learn from the sample code.
+---
+
+# Build an AI chat app with .NET
+
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="openai"
+<!-- markdownlint-enable MD044 -->
+
+Get started with OpenAI and [Semantic Kernel](../semantic-kernel-dotnet-overview.md) by creating a simple .NET 8.0 console chat application. The application will run locally and use the OpenAI `gpt-3.5-turbo` model. Follow these steps to get access to OpenAI and learn how to use Semantic Kernel.
```
Comment (gewarren) [inline docs/ai/quickstarts/get-started-openai.md:19]:
```suggestion
Get started with OpenAI and [Semantic Kernel](../semantic-kernel-dotnet-overview.md) by creating a simple .NET 8 console chat application. The application will run locally and use the OpenAI `gpt-3.5-turbo` model. Follow these steps to get access to OpenAI and learn how to use Semantic Kernel.
```

File: docs/ai/quickstarts/get-started-openai.md [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+
+[!INCLUDE [download-alert](includes/prerequisites-openai.md)]
+
+:::zone-end
+
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="azure-openai"
+<!-- markdownlint-enable MD044 -->
+
+Get started with OpenAI and [Semantic Kernel](../semantic-kernel-dotnet-overview.md) by creating a simple .NET 8.0 console chat application. The application will run locally and connect to the OpenAI `gpt-35-turbo` model deployed into Azure OpenAI. Follow these steps to provision Azure OpenAI and learn how to use Semantic Kernel.
```
Comment (gewarren) [inline docs/ai/quickstarts/get-started-openai.md:29]:
```suggestion
Get started with OpenAI and [Semantic Kernel](../semantic-kernel-dotnet-overview.md) by creating a simple .NET 8 console chat application. The application will run locally and connect to the OpenAI `gpt-35-turbo` model deployed into Azure OpenAI. Follow these steps to provision Azure OpenAI and learn how to use Semantic Kernel.
```

File: docs/ai/quickstarts/get-started-openai.md [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+1. From a terminal or command prompt, navigate to the `azure-openai\02-HikerAI` directory.
+
+2. Use the `dotnet run` command to run the app:
+
+    ```dotnetcli
+    dotnet run
+    ```
+
+    > [!TIP]
+    > If you get an error message, the Azure OpenAI resources may not have finished deploying. Wait a couple of minutes and try again.
```
Comment (gewarren) [inline docs/ai/quickstarts/get-started-openai.md:78]:
```suggestion
    > If you get an error message, the Azure OpenAI resources might not have finished deploying. Wait a couple of minutes and try again.
```

File: docs/ai/quickstarts/get-started-openai.md [hunk 14]
```diff
@@ -0,0 +1,196 @@
...
+
+:::zone-end
+
+:::zone target="docs" pivot="openai"
+
+## Explore the code
+
+The app uses the [`Microsoft.SemanticKernel`](https://www.nuget.org/packages/Microsoft.SemanticKernel) package to send and receive requests to the OpenAI service.
+
+The app code is contained within the **Program.cs** file. The first several lines of code set configuration values and gets the OpenAI Key that was previously set using the `dotnet user-secrets` command.
```
Comment (gewarren) [inline docs/ai/quickstarts/get-started-openai.md:88]:
```suggestion
The app code is contained within the **Program.cs** file. The first several lines of code set configuration values and get the OpenAI Key that was previously set using the `dotnet user-secrets` command.
```

File: docs/ai/quickstarts/get-started-openai.md [hunk 15]
```diff
@@ -0,0 +1,196 @@
...
+
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="azure-openai"
+<!-- markdownlint-enable MD044 -->
+
+## Explore the code
+
+The application uses the [`Microsoft.SemanticKernel`](https://www.nuget.org/packages/Microsoft.SemanticKernel) package to send and receive requests to an Azure OpenAI service deployed in Azure.
+
+The entire application is contained within the **Program.cs** file. The first several lines of code retrieves the secrets and configuration values that were set in the `dotnet user-secrets` for you during the application provisioning.
```
Comment (gewarren) [inline docs/ai/quickstarts/get-started-openai.md:113]:
```suggestion
The entire application is contained within the **Program.cs** file. The first several lines of code retrieve the secrets and configuration values that were set in the `dotnet user-secrets` for you during the application provisioning.
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**